tests/perf/fair_queue_perf.cc

```diff
@@ -52,10 +52,12 @@ struct local_fq_and_class {
         , sfq(sfg, seastar::fair_queue::config())
     {
         fq.register_priority_class(cid, 1);
+        sfq.register_priority_class(cid, 1);
```

Comment:
Without this, it crashes on the first `queue()` call since it requires at least one priority class to work. Most likely a change or refactoring broke this at some point.